addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5964,12 +5977,12 @@ const ImageHandlerOption = SnippetOptionWidget.extend({
     async _initializeImage() {
         return this._loadImageInfo();
     },
-     /**
-     * @private
-     * @param {HTMLImageElement} img
-     * @param {Boolean} [strict=false]
-     * @returns {Boolean}
-     */
+    /**
+    * @private
+    * @param {HTMLImageElement} img
+    * @param {Boolean} [strict=false]
+    * @returns {Boolean}
```

Comment:
This is even wrong formating. The JS doc have to stay like it is (with the space before `*`)